### ISSUE TITLE
No possibility to change default font size used by TableCell.createInnerTable

### ISSUE THREAD

**Issue by funghi123** [dhorions/boxable#154]

**Comment by funghi123** [comment]
It is not possible to:
 - change default font size
 - disable borders

**Comment by Frulenzo** [comment]
You are right with default font size. I will label this issue as feature but what do you mean by disable borders? You have method **removeAllBorders(boolean removeAllBorders)** in Table class

**Comment by funghi123** [comment]
Thank you for response.
Concerning borders - "inner table" created by TableCell in createInnerTable method is always created with borders:

```
		BaseTable table = new BaseTable(yStart, PDRectangle.A4.getHeight() - pageTopMargin, pageTopMargin,
				pageBottomMargin, tableWidth, xStart, doc, currentPage, true, true);
```

**Comment by quodlibetbv** [comment]
ALready supported, see unit tests InnerTableHTMLStylingTest.java and InnerTableStylingTest.java .
